docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -100,7 +100,7 @@ To learn more about how package installation works, see [the conceptual document
 
 ### Get started
 
-To fully onboard your repository you may take the following steps:
+To fully onboard your repository you can do it 2 ways either manually or automated [NuGet.PackageSourceMapper tool](https://www.nuget.org/packages/NuGet.PackageSourceMapper/0.1.3-preview.22218.1#readme-body-tab), for manual onboarding you may take the following steps.

```

Comment:
I'd just create 2 different groups, potentially under subheadings such as `manual onboarding` and `automated` onboarding. 

It's also a good idea to add more details about what the tool deos and how it'd be different from manual onboarding. For example, the tool only adds package ids.

https://devblogs.microsoft.com/nuget/quickly-map-your-nuget-packages-to-sources/#get-started has some good comments, we should try to write a few sentences.

It's also probably a good idea to link https://www.nuget.org/packages/NuGet.PackageSourceMapper#readme-body-tab and not link to a specific version instead.

Reply:
I addressed your comment, please check again.